CONTENT/program.markdown

```diff
@@ -34,9 +34,9 @@ We are going to try hard to make this learning journey exciting for all future w
     [leveldb]( learningMaterials & screencasts ) &
     [regex]( learningMaterials & screencasts )
 
-We encourage you to pick your own projects. Come up with your own or join existing ones. We know, in the beginning this can be challenging, but no worries, we'll help you to explore :-)
+We also encourage all the participants to have some project they want to work on (but if they don't, no worries, we will help them find one or they will be able to join existing projects).
 
-Wizardry is serious business. After each session we will create a simple video summary for you to practice your magic spells at home and collaborate on projects with other wizards, because we believe in collaboration and a learning-by-doing approach.
+Wizardry is serious business. We will create a simple video summary after each session so you can practice your magic spells at home and collaborate with other wizards.
```

Comment:
+1 great.
I wonder if the "learning-by-doing-approach" could be included because i do not think there is another section that could include it instead. What do you think? I kind of tried to bake in everything that goes with our "MANIFESTO" which is not on the page yet, but we might link to it somewhere... The same goes for our "Wall Of Inspiration", but maybe we can do that later on, when we are more sure about the final version of this website text :-)